src/rule_definition_tools.jl

```diff
@@ -236,11 +236,11 @@ function propagation_expr(Δs, ∂s, _conj = false)
     summed_∂_mul_Δs = if n∂s > 1
```

Comment:
It seems the other branch could be removed now.

Reply:
Yes, looks like it